docker-entrypoint.sh

```diff
@@ -16,6 +16,17 @@ if atlantis --help "$1" 2>&1 | grep -q "atlantis $1"; then
     set -- atlantis "$@"
 fi
 
+# If the current uid running does not have a user create one in /etc/passwd
+if ! whoami &> /dev/null; then
+  if [ -w /etc/passwd ]; then
+    echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+  fi
+fi
+
+if  [ $(id -u) != 0 ]; then
```

Comment:
I'd prefer to have a single `exec` at the end of the file. Please add the opposite of this conditional to the `if [ "$1" = 'atlantis' ]` line.

Reply:
Done.